### ISSUE TITLE
[terra-form-checkbox] Incompatible with base 12px font size

### ISSUE THREAD

**Issue by StephenEsser** [cerner/terra-core#2917]
# Feature Request 

The terra-form-checkbox is incompatible with applications that have a base 12px font-size set on the root html element in IE10 or IE 11 in IE 10 compatibility mode. The label is misaligned in Chrome and Safari.

## Description

Applications with a base 12px font-size result in overlap and misalignment of the label and checkbox.

## Steps to Reproduce

1. Clone the terra-core repository
2. Override the html font-size in [terra-base](https://github.com/cerner/terra-core/blob/master/packages/terra-base/src/Base.scss) to 12px.
3. Start the site and navigate to the terra-form-checkbox examples
4. Using an IE 10 browser (Or IE 11 in IE 10 compatibility mode) observe the examples display misaligned and with overlap

## Additional Context / Screenshots

Screenshot taken from IE 10.

![image](https://user-images.githubusercontent.com/6720991/78164526-7be29600-740f-11ea-91c1-8e3118534eda.png)

## Expected Behavior

The checkbox should display appropriately in IE 10 with a base 12 font size.

## Possible Solution

--

## Environment

Browsers: IE 10

Note: The label is misaligned in other browsers. Chrome and Safari.

**Comment by StephenEsser** [comment]
This issue may be resolved by using an appropriate theme that accounts for the reduced base font-size.

**Comment by mjhenkes** [comment]
Long term we should account for font scaling in some fashion and separate base font from the theme all together.

**Comment by neilpfeiffer** [comment]
Suggestion for this would be to resolve sizing layout along with an update the Terra default theme to no longer use the HTML native checkbox and radio button but to have custom styling (originally used native inputs for matching to old Terra, which is no longer needed for default Terra theme -- native inputs are still needed for Fusion theme).

Could combine and complete this existing related Issue: https://github.com/cerner/terra-core/issues/2197

Note - design details need to be added before starting.

**Comment by neilpfeiffer** [comment]
Removing the bug label and changing to enhancement, as changing the rem size for Terra is an application option/possibility, but not fully supported through complete UI validation and testing, 
and will be resolved by including with enhancement #2197.
